app/services/api/endpointService.js

```diff
@@ -49,7 +49,6 @@ angular.module('portainer.services')
   service.createLocalEndpoint = function(name, URL, TLS, active) {
     var endpoint = {
       Name: 'local',
-      URL: 'unix:///var/run/docker.sock',
```

Comment:
Based on my comment above, the payload for the endpoint creation should contain the new `Local` field.